x-pack/test/plugin_api_integration/test_suites/task_manager/task_management.ts

```diff
@@ -338,9 +338,9 @@ export default function ({ getService }: FtrProviderContext) {
 
       await retry.try(async () => {
         const scheduledTask = await currentTask(task.id);
-        expect(scheduledTask.attempts).to.be.greaterThan(0);
+        expect(scheduledTask.attempts).to.be.greaterThan(1);
         expect(Date.parse(scheduledTask.runAt)).to.be.greaterThan(
```

Comment:
Can we run this through the flaky test runner to make sure it's not flaky? I think we've had issues with these types of date based tests before.

Reply:
Yes, I can do that!

Reply:
Here are the flaky test runner results: https://buildkite.com/elastic/kibana-flaky-test-suite-runner/builds/1489